waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -649,7 +650,7 @@
                                               :workingDir home-path}]
                                 :volumes [{:name "user-home"
                                            :emptyDir {}}]
-                                :terminationGracePeriodSeconds 0}}}}
+                                :terminationGracePeriodSeconds 20}}}}
```

Comment:
Should we make this configurable?

Reply:
Created an issue: https://github.com/twosigma/waiter/issues/524